packages/legend-query-builder/style/shared/_query-loader.scss

```diff
@@ -0,0 +1,293 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@use 'mixins' as *;
+
+.query-loader {
+  &__title {
+    padding: 0.3rem;
+    padding-bottom: 2rem;
+  }
+
+  &__filter-section {
+    &__toggler {
+      @include flexVCenter;
+
+      margin-top: 1rem;
+    }
+
+    &__toggler__btn {
+      align-self: flex-start;
+      padding: 0.5rem;
+      display: flex;
+      border-radius: 1rem;
```

Comment:
I think this is really a subjective comment so feel free to disregard but what do you think about lowering the border-radius? We don't have a lot of buttons that are `1rem` apart from the numeric counter ones and it seems that query loader's style is largely sleek so it might be better if it was 0.2. But if its intentional makes sense too, if you wanted to keep it at 1rem to differentiate for user, maybe I would say padding as :0.5rem 0.7rem would give it more space and look clearer though if it's 1rem border-radius.